cmd/zetaclientd/start.go

```diff
@@ -214,16 +213,16 @@ func start(_ *cobra.Command, _ []string) error {
 		return err
 	}
 
-	bitcoinChainID := chains.BitcoinRegtest.ChainId
-	btcChain, _, btcEnabled := appContext.GetBTCChainAndConfig()
-	if btcEnabled {
-		bitcoinChainID = btcChain.ChainId
+	btcChain, err := appContext.FirstChain(zctx.Chain.IsUTXO)
```

Comment:
FirstChain sounds ambiguous. Consider renaming this to FilterChains.

Reply:
https://github.com/zeta-chain/node/pull/2568/commits/f2e3facc2e68022c857b3b981dc2366917268029